src/Common/XDBCBridgeHelper.h

```diff
@@ -109,7 +109,8 @@ class XDBCBridgeHelper : public IXDBCBridgeHelper
             uri.setPath(IDENTIFIER_QUOTE_HANDLER);
             uri.addQueryParameter("connection_string", getConnectionString());
 
-            ReadWriteBufferFromHTTP buf(uri, Poco::Net::HTTPRequest::HTTP_POST, nullptr);
+            ReadWriteBufferFromHTTP buf(
+                uri, Poco::Net::HTTPRequest::HTTP_POST, {}, ConnectionTimeouts(http_timeout, http_timeout, http_timeout));
```

Comment:
Maybe we should use `ConnectionTimeouts::getHTTPTimeouts(context)`?

Reply:
Done.